src/platform/utilities/tests/feature-toggles/helpers.unit.spec.js

```diff
@@ -0,0 +1,83 @@
+import { expect } from 'chai';
+
+const {
+  generateToken,
+  getFlipperId,
+} = require('../../feature-toggles/helpers');
+
+describe('feature toogles', () => {
+  let oldCookie;
+
+  before(() => {
+    oldCookie = document.cookie;
+    document.cookie = '';
+  });
+
+  describe('generateToken', () => {
+    it('should return a string', () => {
+      document.cookie = '';
+      expect(generateToken()).to.be.a('string');
+    });
+
+    it('should return a not-null value', () => {
+      document.cookie = '';
+      expect(generateToken()).to.not.be.null;
+    });
+
+    it('should return a not-empty value', () => {
+      document.cookie = '';
+      expect(generateToken()).to.not.be.empty;
+    });
+  });
+
+  describe('getFlipperId', () => {
+    it('should set the FLIPPER_ID cookie to a string', () => {
+      document.cookie = '';
+      expect(getFlipperId()).to.be.a('string');
+    });
+
+    it('should set the FLIPPER_ID cookie to not be null', () => {
+      document.cookie = '';
+      expect(getFlipperId()).to.not.be.null;
+    });
+
+    it('should set the FLIPPER_ID cookie to not be empty', () => {
+      document.cookie = '';
+      expect(getFlipperId()).to.not.be.empty;
+    });
+
+    it('should set the FLIPPER_ID cookie to a non-empty string', () => {
+      document.cookie = '';
+      getFlipperId();
+      expect(
+        document.cookie.replace(
+          /(?:(?:^|.*;\s*)FLIPPER_ID=\s*([^;]*).*$)|^.*$/,
+          '$1',
+        ),
+      ).to.be.a('string');
+      expect(
+        document.cookie.replace(
+          /(?:(?:^|.*;\s*)FLIPPER_ID=\s*([^;]*).*$)|^.*$/,
+          '$1',
+        ),
+      ).to.not.be.null;
+      expect(
+        document.cookie.replace(
+          /(?:(?:^|.*;\s*)FLIPPER_ID=\s*([^;]*).*$)|^.*$/,
+          '$1',
+        ),
+      ).to.not.be.empty;
```

Comment:
```suggestion
      const cookie = document.cookie.replace(
         /(?:(?:^|.*;\s*)FLIPPER_ID=\s*([^;]*).*$)|^.*$/,
         '$1',
       );
      expect(cookie).to.be.a('string');
      expect(cookie).to.not.be.empty;
```
We can grab the cookie just once and then test it.
Also, if it's a string, it's not going to be null.